docs/sources/mimir/configure/configure-zone-aware-replication.md

```diff
@@ -87,6 +89,16 @@ Deploying Grafana Mimir with zone-aware replication across multiple cloud provid
 
 The [Kubernetes Rollout Operator](https://github.com/grafana/rollout-operator) is a Kubernetes operator that makes it easier for you to manage multi-availability-zone rollouts. Consider using the Kubernetes Rollout Operator when you run Grafana Mimir on Kubernetes with zone awareness enabled.
 
+Note that even in a scenario where you have deployed Grafana Mimir to a single physical availability zone, the Rollout Operator is immensely useful.
+
+This is because it automates and accelerates the process of rollouts by:
+
+- Allowing an entire zone to be updated at a time
+- Prioritizing rollout of unhealthy zones over healthy zones to ensure stability
+- Allowing the administrator to determine how aggressive rollouts should be
+
+This functionality is particularly useful in larger deployments where rollouts would otherwise take a very long time and likely require manual intervention if issues were to occur.
```

Comment:
```suggestion
The automated functionality provided by an Operator is particularly useful in larger deployments where rollouts would otherwise take a very long time and likely require manual intervention if issues were to occur.
```